.github/workflows/pr_review_reminder.yml

```diff
@@ -0,0 +1,21 @@
+name: PRs reviews reminder
+
+on:
+  schedule:
+    # Every weekday every 2 hours during working hours, send notification
+    - cron: "0 8-17/2 * * 1-5"
+  workflow_dispatch:
```

Comment:
Is this needed if it doesn't have any values?

Reply:
yes its enabling the ppl to be run manually, and it does not have params